src/agent.ts

```diff
@@ -0,0 +1,98 @@
+import { Finding, HandleTransaction, TransactionEvent, FindingSeverity, FindingType } from "forta-agent";
+import {
+  DEPLOYER_ADDRESS,
+  DEPLOYMENT_TO_ADDRESS,
+  CREATE_AGENT_SIG,
+  UPDATE_AGENT_SIG,
+  DISABLE_AGENT_SIG,
+} from "./constants";
+
+const createFinding = (
```

Comment:
A team practice for us to put the alert/finding creation functions in their own `findings.ts` files. Since this is a simple bot, this is fine, but worth keeping in mind going forward.